core/src/main/scala/io/snappydata/SnappyStoreTableSizeProviders.scala

```diff
@@ -88,20 +84,14 @@ object StoreTableValueSizeProviderService extends Logging {
     if (currentTableSizeInfo == null || !currentTableSizeInfo.contains(tableName)) {
```

Comment:
@nthanvi the contains check is redundant here with the below pattern match for get